frontend/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,46 @@
+type TagProps = {
+  /** Content of tag */
+  children: React.ReactNode;
+
+  /** href destination Link */
+  to: String;
+
+  /** Possibility to change color and background of tag */
+  color?:
```

Comment:
Wenn color optional ist (das macht das ? am Ende) => Einen default setzen. Das geht unten `React.FC<TagProps> = ({ children, to, color = "DEFAULT_VALUE" }) => {`. Oder color nicht optional machen :D